dracula-theme.jsonc

```
@@ -6,35 +6,49 @@
   "themeValues": {
     // values applied to the entire app
     "root": {
+      "current__line": "#44475a",
+      "background": "#282a36",
+      "foreground": "#f8f8f2",
+      "comment": "#6272a4",
+      "pink": "#ff79c6",
       "red": "#ff5555",
       "orange": "#ffb86c",
       "yellow": "#f1fa8c",
       "green": "#50fa7b",
-      "purple": "#bd93f9",
-      "pink": "#ff79c6",
-      "cyan": "#8be9fd",
-      "app__bg0": "#282a36",
-      "comment": "#6272a4",
+      "blue": "#bd93f9",
+      "ltblue": "#8be9fd",
+      "app__bg0": "lighten(@background, 5%)",
       "toolbar__bg0": "lighten(saturate(@app__bg0, 3%), 6%)",
       "toolbar__bg1": "lighten(@toolbar__bg0, 4%)",
       "toolbar__bg2": "lighten(@toolbar__bg1, 6%)",
-      "panel__bg0": "lighten(@app__bg0, 5%)",
+      "panel__bg0": "lighten(@background, 5%)",
       "panel__bg1": "lighten(@panel__bg0, 5%)",
       "panel__bg2": "lighten(@panel__bg1, 5%)",
-      "input__bg": "fade(@app__bg0, 20%)",
+      "input__bg": "fade(@background, 10%)",
```

Why the fade percent change? Unless there is a good reason we should keep it consistent.
```suggestion
      "input__bg": "fade(@background, 20%)",
```